go/plugins/googleai/googleai.go

```diff
@@ -522,8 +547,6 @@ func translateCandidate(cand *genai.Candidate) *ai.ModelResponse {
 
 //copy:stop
 
-//copy:start vertexai.go translateResponse
```

Comment:
VertexAI should not replicate this function anymore since `u.CachedContentTokenCount` (@ line 558) is not being provided in the response struct (see https://pkg.go.dev/cloud.google.com/go/vertexai/genai#UsageMetadata)